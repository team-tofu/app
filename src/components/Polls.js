import React, { PureComponent } from 'react';

const polls = [
  { question: 'some question', options: ['option1', 'option2'] },
  { question: 'some question', options: ['option1', 'option2'] },
  { question: 'some question', options: ['option1', 'option2'] },
  { question: 'some question', options: ['option1', 'option2'] }
];

export default class Polls extends PureComponent {
  render() {
    const pollList = polls.map((poll, i) => {
      return <li key={i}>{poll.question}</li>;
    });
    return (
      <ul>
        {pollList}
      </ul>
 
    );

  }
}
