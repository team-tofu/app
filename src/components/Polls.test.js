import React from 'react';
import renderer from 'react-test-renderer';
import Polls from './Polls';
describe('Polls Test', () => {
  it('Renders list of polls', () => {

    const tree = renderer.create(
      <Polls /> 
    );
    expect(tree).toMatchSnapshot();
  });
});
