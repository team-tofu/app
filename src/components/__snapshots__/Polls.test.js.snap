// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Polls Test Renders list of polls 1`] = `
<ul>
  <li>
    some question
  </li>
  <li>
    some question
  </li>
  <li>
    some question
  </li>
  <li>
    some question
  </li>
</ul>
`;
